tests/cross_fw/test_templates/helpers.py

```diff
@@ -466,3 +466,22 @@ def __init__(self):
 
     def forward(self, x):
         return self.conv(x)
+
+
+class RoPEModel(nn.Module):
```

Comment:
I wasn't able to clearly generate the PT graph to properly visualise the pattern variant. I've created a model based on this interpretation (https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L149-L169) since I have no others.
@alexsu52

Reply:
Thanks, looks good.